description.md

```diff
@@ -3,11 +3,44 @@
 This file contains all the topics that are covered by the Open Education Hub methodology.
 Each topic in this file contains a small description and links to the files cover it more thouroughly.
 
-
 ## Teaching Stages
 
 ### Lectures
 
+Lectures are teaching activities that typically take place with a larger audience and are asymmetrical: a lecturer mostly talks while the audience mostly listens.
+Interaction is (quite obviously) important, but it won't scale to a larger audience and, because the entire communication medium is shared, is limited to on conversation only.
+Labs, for example, allow each participant to focus on one item they could work on.
+Seminars are close to lectures, except that they typically involve a smaller group of people and are naturally interactive, with the relationship being less asymmetrical.
```

Comment:
I think this build-up does not lead anywhere, consider talking about labs and seminars to introduce the need for materials (in multiple forms) to make lectures as effective.

Reply:
On a second thought, you could just rephrase this to: "Lectures are different from practical sessions (link to practical sessions) by ...".

Reply:
@Alex-deVis , @RazvanN7 , indeed. I rephrased this.